dev/create-release/release-build.sh

```diff
@@ -53,7 +53,7 @@ if [ $# -eq 0 ]; then
   exit_with_usage
 fi
 
-if [[ $@ == *"help"* ]]; then
+if [[ "$*" == *"help"* ]]; then
```

Comment:
Likewise what is the difference here

Reply:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).